modules/n1ql/pages/n1ql-language-reference/curl.adoc

```diff
@@ -526,7 +528,7 @@ LIMIT 4;
 Note that the results from the `CURL()` output are embedded in the `results[]` array under the keyspace alias `t` used in the remote query.
 So, we extract the result documents appropriately with the expression `+CURL(...).results[*].t+` and alias it to `route` as the left side keyspace for the `JOIN`.
 
-RBAC credentials are required when CURL() is accessing Couchbase Server version 5.0 or later.
+RBAC credentials are required.
```

Comment:
Adding reference to Capella back in, because RBAC credentials won't mean anything when accessing external sources.

```suggestion
RBAC credentials are required when CURL() is accessing Couchbase Capella.
```